cfg/1.11/master.yaml

```diff
@@ -366,15 +366,14 @@ groups:
     scored: true
 
   - id: 1.1.23
-    text: "Ensure that the --service-account-lookup argument is set to true (Scored)"
+    text: "Ensure that the --service-account-lookup argument is not set to false (Scored)"
```

Comment:
@jnummelin please keep the original `text` for this check for compatibility with the CIS benchmark.